NSS-Day-Cohort-42/rare-API-media-moguls
# Rp create tag

#### Changes Made
1. created fixture file for tags with seed data
1. fixed key variable name in rare user model to match erd and ran makemigrations
1. created a view file for tag with a create tag method and serializer
1. added reference to tag view in urls 
1. added list method to get all tags in view
​
#### Steps to Review
1. git fetch --all
1. git checkout rp-createTag
```

3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> run python manage.py migrate 
> run python manage.py loaddata for fixtures files
>test that a new tag can be created in postman and get all tags 
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
